testutil/keeper/proof.go

```diff
@@ -76,7 +94,12 @@ func ProofKeeper(t testing.TB) (keeper.Keeper, context.Context) {
 	return k, ctx
 }
 
-func NewProofKeeperWithDeps(t testing.TB) (ProofKeeperWithDeps, context.Context) {
+// NewProofModuleKeepers is a helper function to create a proof keeper and a context. It uses
+// real dependencies for all keepers except the bank keeper, which is mocked as it's not used
+// directly by the proof keeper or its dependencies.
+//
+// TODO_CONSIDERATION: can we remove the bank keeper as a dependency of the proof keeper?
```

Comment:
What information do you feel like you're lacking for this?

Reply:
This is now done as of #406 .

Reply:
So we should remove the comment, right?